src/scoring.go

```diff
@@ -102,19 +106,41 @@ func scoreChecks() {
 
 	for _, check := range mc.Config.Check {
 		status := true
+		passStatus := []bool{}
 		for _, condition := range check.Pass {
-			status = processCheckWrapper(&check, condition.Type, condition.Arg1, condition.Arg2, condition.Arg3)
-			if reverseEnabled {
-				status = !status
+			passItemStatus := processCheckWrapper(&check, condition.Type, condition.Arg1, condition.Arg2, condition.Arg3)
+			if debugEnabled {
+				infoPrint(fmt.Sprint("Result of last pass check was ", status))
 			}
-			if status {
+			passStatus = append(passStatus, passItemStatus)
+		}
+
+		// For multiple pass conditions, will only be true if ALL of them are
+		for _, result := range passStatus {
+			status = status && result
+			if !status {
+				break
+			}
+		}
+		if debugEnabled {
+			infoPrint(fmt.Sprint("Result of all pass check was ", status))
```

Comment:
`infoPrint("Result of all pass check was " + status)`

Reply:
I don't think you can concat strings and bools like that. It is ugly with `Sprint` but I can't think of any cleaner solutions.